cvat/apps/dataset_manager/tests/test_streaming_efficiency.py

```diff
@@ -198,86 +208,46 @@ def test_can_stream_efficiently_on_export(self):
             else:
                 item_ids = {("image", "foo"), ("another_image", "foo")}
             with self.subTest(data_cls=data_cls.__name__):
-                instance_data = self._make_mock_instance_data(data_cls, item_ids)
-                extractor_cls = (
-                    CVATProjectDataExtractor
-                    if data_cls is ProjectData
-                    else CvatTaskOrJobDataExtractor
-                )
-                extractor_cls = self._make_counting_data_extractor_cls(extractor_cls)
+                extractor_cls = self._make_counting_data_extractor_cls()
 
                 if data_cls in [JobData, TaskData]:
```

Comment:
It seems that the task and job tests are checking the same now, shouldn't they be split into 2 different cases?